src/scss/_base.scss

```diff
@@ -73,7 +66,48 @@ li + li {
   color: var(--color-red);
 }
 
-*:focus {
-  outline: 3px dashed #000;
-  outline-offset: 2px;
+//* Focus set up *//
+
+:is(a, button, input, textarea, summary) {
```

Comment:
Checked browser compatibility of :is and it shouldn't cause any issues - if someone uses an older version of the browser that doesn't support :is, it shouldn't be an issue too (as far as I understand) as focus styles will fall back to the browser's default